ws-nextjs-app/pages/[service]/live/[id]/__snapshots__/live.test.tsx.snap

```diff
@@ -1159,14 +1160,15 @@ exports[`Live Page creates snapshot of the live page 1`] = `
             <span
               aria-hidden="true"
               class="emotion-4"
+              data-testid="live-label"
             >
-              LIVE
+              Live Coverage
             </span>
             <span
               class="emotion-5"
               lang="en-GB"
             >
-              Live, 
+              Live Coverage ,
```

Comment:
Is this meant to have a space before the comma?

Reply:
Good spot! It should be the other way round.